test/bats/azure-test.bats

```diff
@@ -50,6 +50,7 @@ SLEEP_TIME=1
 }
 
 @test "cosign test" {
+    skip "Skipping test for now as cosign verifier plugin does not support workload identity yet."
```

Comment:
we can turn this back on. The cosign auth PR will most likely go in before this one.

Reply:
looks like it's merged, I'll turn it on.